src/AspNet.Security.OAuth.EVEOnline/EVEOnlineAuthenticationDefaults.cs

```diff
@@ -31,45 +31,41 @@ public static class EVEOnlineAuthenticationDefaults
     /// </summary>
     public static readonly string CallbackPath = "/signin-eveonline";
 
-    /// <summary>
-    /// Default values for the Tranquility (live) server.
-    /// </summary>
```

Comment:
Any particular reason you deleted this?

Reply:
Believe this was most likely just being overzealous with the rebase fixing.
Will add them back.